styles/Canonical/012-Date-format.yml

```
@@ -0,0 +1,15 @@
+extends: existence
+message: Use the '1 January 1970' format instead of '%s'
+scope: raw
```

```suggestion
message: Use the '1 January 1970' format instead of '%s'
link: https://docs.ubuntu.com/styleguide/en/#dates
scope: raw
```

Since other rules add the link to the relevant heading, I suggest adding it for this one as well.